ambari-metrics/ambari-metrics-timelineservice/pom.xml

```diff
@@ -738,6 +737,25 @@
       <version>3.2</version>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
```

Comment:
Think about reducing the scope of the dependencies maybe to 'provided'.
Also you could add comments to elaborate on why the dependencies are required.

Reply:
I will delete these because these jars are used only distributed mode.